### PR TITLE
Add support for CMEK keys in Data Foundation end to end example

### PR DESCRIPTION
**Pull request by lcaggio**
Add support for CMEK keys in Data Foundation end to end example.

### CODE REVIEW

**Comment by ludoo** [comment]
Lore, did not have time for a review, but are keys completely optional? What happens if the user does not set the corresponding variable?

**Comment by yorambenyaacov** [comment]
project-datamart in 01-environment is missing service_encryption_key_ids for bigtable

File: data-solutions/data-platform-foundations/01-environment/variables.tf [hunk 1]
```diff
@@ -55,3 +55,17 @@ variable "service_account_names" {
...
+variable "service_encryption_key_ids" {
+  description = "Cloud KMS encryption key in {LOCATION => [KEY_URL]} format. Keys belong to existing project."
+  type = object({
+    regional      = string
+    multiregional = string
+    global        = string
+  })
+  default = {
+    regional      = null
+    multiregional = null
```
Comment (yorambenyaacov) [inline data-solutions/data-platform-foundations/01-environment/variables.tf:68]: the code is using only multiregional.
1. Do we need the other options?
2. If yes (I assume we need it), "how to use" example can be nice.

Comment (lcaggio) [reply data-solutions/data-platform-foundations/01-environment/variables.tf:68]: Regional key not in use at the moment, removed.

**Pull request merged**